packages/bpk-component-infinite-scroll/README.md

```diff
@@ -162,13 +162,14 @@ Updates the internal array and triggers all listeners.
 
 ## Props
 
-| Property                | PropType               | Required | Default Value |
-| ----------------------- | ---------------------- | -------- | ------------- |
-| dataSource              | instanceOf(DataSource) | true     | -             |
-| elementsPerScroll       | number                 | false    | 5             |
-| initiallyLoadedElements | number                 | false    | 5             |
-| onScroll                | func                   | false    | null          |
-| onScrollFinished        | func                   | false    | null          |
-| renderLoadingComponent  | func                   | false    | null          |
-| renderSeeMoreComponent  | func                   | false    | null          |
-| seeMoreAfter            | number                 | false    | null          |
+| Property                | PropType                         | Required | Default Value |
+| ----------------------- | -------------------------------- | -------- | ------------- |
+| dataSource              | instanceOf(DataSource)           | true     | -             |
+| elementsPerScroll       | number                           | false    | 5             |
+| initiallyLoadedElements | number                           | false    | 5             |
+| loaderMinDisplay        | oneOf(['small', 'half', 'full']) | false    | 'full'        |
```

Comment:
Not a big fan of the name. It doesn't really indicate what it does IMO. Can we maybe use something like `loaderIntersectionTrigger`? with values like initial, half, all/full?

Reply:
I was not very proud of the name either, `loaderIntersectionTrigger` sounds better.
`initial` is not accurate because the intersector uses percentages and if for some reason you start displaying the element from below it will trigger as well, that's why small instead